### ISSUE TITLE
The image quality has changed, and the image has a ripple.

### ISSUE THREAD

**Issue by martinTangky** [bumptech/glide#820]
I try to load them as 8888 but it's not use.
 if i load it by xml ,The image don't has a ripple

**Comment by martinTangky** [comment]
![3215312](https://cloud.githubusercontent.com/assets/10938249/11804168/aee6ad3c-a33c-11e5-9afe-5185e9dd997a.png)

**Comment by martinTangky** [comment]
The original  
![test2](https://cloud.githubusercontent.com/assets/10938249/11804194/e3f537be-a33c-11e5-97a1-b620fbda6947.jpg)

**Comment by TWiStErRob** [comment]
This sounds like #815. If you load it in XML (I assume you mean android:src) or as `.placeholder()` it is loaded through android resources, so Glide is not really involved; but `.load()` uses Glide's configuration and decoders (mostly `BitmapFactory`).

You can [change to 8888 globally](https://github.com/bumptech/glide/wiki/Configuration#bitmap-format) or per load: `.asBitmap().format(PREFER_ARGB_8888)`.

**Comment by martinTangky** [comment]
I  load it in XML just wanna test this image, i also tryed to loaded like :.asBitmap().format(PREFER_ARGB_8888) but it`s don`t work

**Comment by TWiStErRob** [comment]
Can you please share your XML and Glide load line?

*(please try to use ' instead of ` when writing possessives or contractions (X's, it's, doesn't, etc), the slanted tick confuses GitHub's interpreter, that's reserved for displaying code if you surround something with it.)*

**Comment by martinTangky** [comment]
`Glide.with(context).load(AppConstance.netHead + sNewsList.get(position).get_images800()).asBitmap()
				.format(DecodeFormat.ALWAYS_ARGB_8888).into(h.img);`

**Comment by TWiStErRob** [comment]
Hmm, isn't "always" deprecated? try "prefer".

**Comment by martinTangky** [comment]
I've already tried to use "prefer",but  it's not work too. T.T

**Comment by martinTangky** [comment]
Like This image ,if i use the 'Picasso' to load my image,it will not happen.
As i kown,'Picasso' is use ARGB_8888 by default .

**Comment by TWiStErRob** [comment]
I was able to reproduce your problem like this:
```java
Glide
	.with(context)
	.load(url)
	.asBitmap()
	//.format(DecodeFormat.PREFER_ARGB_8888)
	.into(imageView);
```
and then launching the app (from source: nice picture), closing it, and reopening it (from cache: banded picture). As soon as I uncommented the `format` line and did the same launch-close-reopen it loaded nicely both times. You shouldn't even need to clear the cache between these changes because it's part of the cache key.
